Numsense/Chinese.fs

```diff
@@ -0,0 +1,194 @@
+﻿module internal Ploeh.Numsense.Chinese
+
+open Ploeh.Numsense.InternalDsl
+
+type CharacterSet =
+    {
+        negative: string;
```

Comment:
Since this is an internal type, it may not be that big a deal, but since `CharacterSet` is a type, it'd be more idiomatic to give each named element a PascalCased name: `Negative`, `Zero`, `One`, etc.

From an API design perspective, it doesn't matter, because the type (currently) isn't exposed to the public, but from a readability perspective, it constitutes a small 'speed bump' for the reader of the code.